### PR TITLE
Issue 64: removal of DoesExist

### PR DESCRIPTION
**Pull request by matiwinnetou**

### CODE REVIEW

File: demos/hello/hello.go [hunk 1]
```diff
@@ -63,8 +63,10 @@ func (c *ScaleSystem) Update(e *engi.Entity, dt float32) {
 	var mod float32
 
 	if rand.Int()%2 == 0 {
+		render.Priority = engi.Hidden
```
Comment (paked) [inline demos/hello/hello.go]: I'd prefer to keep the `hello` example *only* doing things with scale.

**Comment by matiwinnetou** [comment]
@paked  @EtienneBruines please review again after applying fixes.

**Comment by paked** [comment]
@matiwinnetou This is ready to merge correct?

**Comment by matiwinnetou** [comment]
Please merge.

**Comment by EtienneBruines** [comment]
LGTM. @paked can merge when he agrees.

**Comment by paked** [comment]
Thanks for the contribution @matiwinnetou

**Pull request merged**